scene/gui/file_dialog.cpp

```diff
@@ -127,6 +127,7 @@ void FileDialog::_dir_entered(String p_dir) {
 
 	dir_access->change_dir(p_dir);
 	file->set_text("");
+	file->emit_signal("text_changed", "");
```

Comment:
I don't understand, LineEdit already trigger the event, right? Why do you call it again?

Reply:
https://github.com/godotengine/godot/commit/2abec59db96496611bb16dd1300d9b7d3def9780

This is probably not needed anymore after this was added https://github.com/godotengine/godot/issues/19372

Reply:
Didn't know that LineEdit doesn't emit signal when changed by code.